src/main/java/frc/robot/commands/conveyor/ConveyorStop.java

```diff
@@ -47,7 +46,11 @@ public void initialize()
   @Override
   public void execute()
   {
-    robot.conveyor.stop();
+    if (robot.intake.spinning) {
+      robot.conveyor.feedSlow();
```

Comment:
What if you already have a ball in the conveyor and want to intake a second ball?

Reply:
Also what if the intake is outaking?  Does this still spin the conveyor?

Reply:
There's a deadzone in the robot between the conveyor and the shooter so it doesn't matter if the other ball keeps getting pushed. And it doesn't spin the conveyor automatically when outaking.